OpenRA.Mods.Cnc/Graphics/ClassicTilesetSpecificSpriteSequence.cs

```diff
@@ -34,20 +34,56 @@ public class ClassicTilesetSpecificSpriteSequence : ClassicSpriteSequence
 		[Desc("Dictionary of <tileset name>: filename to override the Filename key.")]
 		static readonly SpriteSequenceField<Dictionary<string, string>> TilesetFilenames = new SpriteSequenceField<Dictionary<string, string>>(nameof(TilesetFilenames), null);
 
-		public ClassicTilesetSpecificSpriteSequence(ModData modData, string tileset, SpriteCache cache, ISpriteSequenceLoader loader, string image, string sequence, MiniYaml data, MiniYaml defaults)
-			: base(modData, tileset, cache, loader, image, sequence, data, defaults) { }
+		public ClassicTilesetSpecificSpriteSequence(SpriteCache cache, ISpriteSequenceLoader loader, string image, string sequence, MiniYaml data, MiniYaml defaults)
+			: base(cache, loader, image, sequence, data, defaults) { }
 
-		protected override string GetSpriteFilename(ModData modData, string tileset, string image, string sequence, MiniYaml data, MiniYaml defaults)
+		protected override IEnumerable<(string Filename, int[] loadFrames, int[] useFrames, MiniYamlNode.SourceLocation location)> ParseFilenames(ModData modData, string tileset, int[] frames, MiniYaml data, MiniYaml defaults)
```

Comment:
I'm sure I mentioned it in another file, but inconsistent naming - `loadFrames`, `useFrames` and `location` should be PascalCase like `Filename`.